server/app/views/admin/programs/ProgramImageView.java

```diff
@@ -172,12 +189,45 @@ private StorageUploadRequest createStorageUploadRequest(ProgramDefinition progra
     return publicStorageClient.getSignedUploadRequest(key, onSuccessRedirectUrl);
   }
 
-  private DivTag renderCurrentImage(ProgramDefinition program) {
-    if (program.summaryImageFileKey().isEmpty()) {
-      return div();
+  private DivTag renderCurrentProgramCard(Http.Request request, ProgramDefinition program) {
+    DivTag currentProgramCardSection =
+        div().with(h2("What the applicant will see").withClasses("mb-4"));
+
+    Optional<CiviFormProfile> profile = profileUtils.currentUserProfile(request);
+    Long applicantId;
+    try {
+      applicantId = profile.get().getApplicant().get().id;
+    } catch (NoSuchElementException | ExecutionException | InterruptedException e) {
+      return currentProgramCardSection.with(p("Applicant preview cannot be rendered"));
```

Comment:
suggestion (non-blocking):  I wonder if we should also log a warning here for troubleshooting with a more specific message like "Could not fetch applicant id".

Reply:
Agreed, done!